src/app/wallet/overview/widgets/stake/stake.component.html

```diff
@@ -3,7 +3,7 @@
     <div class="subtitle">Staking</div>
   </header>
 
-  <mat-card *ngIf="_stake.stakingEnabled && _stake.isStaking && !isLocked()" class="staking-node active">
+  <mat-card *ngIf="_stake.stakingEnabled && !isLocked()" class="staking-node active">
```

Comment:
how is stakeingEnabled set?

Reply:
Staking enabled is fetched by stake service and observing getstakinginfo rpc